config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
-      resources :articles, only:[:index, :show]
+      resources :articles, only:[:index, :create]
+      post 'articles/create'
```

Comment:
Try to remove this line and see if it work. I think we should have the post route open

Reply:
We removed it and the tests still passed. 
So we are going to keep it out for the final pr.